cura/PrinterOutput/PrinterOutputModel.py

```diff
@@ -90,6 +90,10 @@ def homeHead(self):
     def homeBed(self):
         self._controller.homeBed(self)
 
+    @pyqtSlot(str)
+    def sendCustomCommand(self, command: str):
```

Comment:
How about "sendRawCommand" or "sendGcodeCommand" instead? Slightly more reusable in (other) plugins

Reply:
I really like the idea of sending Raw G-code only it is such a master mode which might break machine by making small typo.

Reply:
It would be great if this functionality could be a downloadable plugin.

Reply:
Right now that would mean overwriting the entire monitor stage? There is no QML target area to place a button there.

Reply:
The proper way to do this would probably involve creating a model of monitor plugins that can each provide a QML file.

Reply:
Renamed to sendRawCommand. The other thing I can't really do until there is a QML target area for it.